### DISCUSSION TITLE
Git gor Android aaps studio

### DISCUSSION THREAD

**Discussion by MatlM11** [community/community#140930] [Programming Help]
Please I tried to reinstall andriod studio a build aaps aplication, but during the building process (grt bcs code) appears note that git is not available. I know that is not such detail, but coukd you help, first time eas enough I checjed git batch instslled.  Or please ehere can I find any help? Thank you Martin

**Comment by awesmubarak** [comment]
You can check if git is installed from your terminal using `git --version`. If it isn't, install it from the [website](https://git-scm.com/downloads). If it is installed, in android studio check the status using `File > Settings > Version Control > Git`, make sure that the git path is properly set. Hope this helps.

**Reply by MatlM11** [reply]
Thank you, reinstalled several times , but from exe file on computer . I will check the setting, I am not sure about the URL.  Tnank you

**Comment by queenofcorgis** [comment]
Hey there!  👋 

Thanks for posting in the GitHub Community, @MatlM11 ! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category. The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them.

I've gone ahead and moved this to the correct category for you. Good luck!
